modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -46,8 +46,8 @@ export class {{classname}} {
 
 {{#operation}}
     /**
-     * {{summary}}
      * {{notes}}
+     {{#summary}}* @summary {{&summary}}{{/summary}}
```

Comment:
Given that `summary` is not a required field in the spec, I think the above may result in empty line with 4 leading spaces. What about using the following instead?
```
    {{#summary}}
    * @summary {{&summary}}
    {{/summary}}
```